### DISCUSSION TITLE
why when i use fooocus colab the models doesn't work

### DISCUSSION THREAD

**Discussion by alwadeai1** [lllyasviel/Fooocus#3765] [General]
when i use fooocus colab and dowenload modesl or loras they don't show in fooocus 
![Screenshot 2024-12-01 235511](https://github.com/user-attachments/assets/82f3f9dc-94f0-420f-be4a-4998515d496b)
![Screenshot 2024-12-01 235547](https://github.com/user-attachments/assets/a9c8ecb6-5ce4-437d-b389-ebd40b5ff42e)

**Comment by poor7** [comment]
Rename the downloaded file to something normal, for example Jug11.safetensors

**Reply by alwadeai1** [reply]
i'll try

**Reply by alwadeai1** [reply]
it doesn't work

**Reply by poor7** [reply]
Did you move the downloaded and renamed file to the /Fooocus/models/checkpoints/ folder?

**Reply by alwadeai1** [reply]
it doesn't work 
could you try it in your colab to be sure if the problem form my internet conaction

**Reply by alwadeai1** [reply]
thank for your help but solve it i just have to add ( .safetensors ) after the name
